app/events/views.py

```diff
@@ -115,19 +115,14 @@ def post():
 
                 for event_res in event_models:
                     var = str(request.data.get(event_res, '').capitalize())
+                    var = var.strip(' \t\n\r')
                     if not var:
                         response = {
                             "message":'{} missing'.format(event_res)
                         }
                         return make_response(jsonify(response)), 401
                     args.update({event_res:var})
 
-                if args['title'][0] is  " ":
-                    response = {
-                        "message":'Event name cannot contain a space for the first letter'
-                    }
-                    return make_response(jsonify(response)), 401
```

Comment:
the best status code for validation is 400...bad request.

Reply:
@betsybeth  that piece of code is deleted.
I'll refactor my active code to this review